client/reader/following-manage/index.jsx

```diff
@@ -97,6 +103,7 @@ class FollowingManage extends Component {
 					<h1>{ translate( 'Manage Followed Sites' ) }</h1>
 				</MobileBackToSidebar>
 				{ searchResults.length === 0 && <QueryReaderFeedsSearch query={ sitesQuery } /> }
+				{ recommendedSites.length === 0 && <QueryReaderRecommendedSites seed={ this.state.seed } /> }
```

Comment:
I didn't think none could be found so I didn't include the null state for unfetched...